crim/models/user.py

```diff
@@ -22,6 +22,7 @@ class Meta:
         'CRIMPerson',
         on_delete=models.SET_NULL,
         null=True,
+        blank=True,
```

Comment:
We don't want both `null=True` and `blank=True`. The first should be sufficient here.